frontend/src/features/projects/common/ProjectRouter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
TODO: eventually I'd like to put the update the dispose workflow and remove the /dispose part of the route so that the frontend 1-1 matches the status routes in the DB. Leaving that out for now due to time constraints.